### PR TITLE
Bug 1267573 update webapp bug bounty faq

### PR DESCRIPTION
**Pull request by jpetto**
## Description

Clean up commit on original PR (#4135).

## Bugzilla link

[bug 1267573](https://bugzilla.mozilla.org/show_bug.cgi?id=1267573)

## Testing

Only HTML updates. Make sure no typos or other tag/structure releated errors.

### CODE REVIEW

**Comment by craigcook** [comment]
r+ 🐛

File: bedrock/security/templates/security/bug-bounty/faq-webapp.html [hunk 1]
```diff
@@ -113,34 +112,150 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
...
-          <p>The Mozilla Developer Network (developer.mozilla.org) has been removed from this
-            list due to site vandalism by researchers looking for security issues. It will be
+          <p>Below is the list of domains eligible for Mozilla’s website and services
+            security bug bounty. Please check it regularly, as it changes over time.</p>
+
+          <h4>www.mozilla.com/org</h4>
+          <ul>
+          	<li>www.mozilla.com</li>
+          	<li>www.mozilla.org</li>
+          	<li>www.firefox.com<li>
```
Comment (craigcook) [inline bedrock/security/templates/security/bug-bounty/faq-webapp.html]: D'oh, I missed that this was still missing a correct `</li>`

Comment (jpetto) [inline bedrock/security/templates/security/bug-bounty/faq-webapp.html]: :rage3:

/me falls down

Fixed directly on master in d7580090e46a0fba598a1f096c9c1c76c466708a.

**Comment by amuntner** [comment]
Thank guys, I was going to fixup this evening. Much appreciated.

**Pull request merged**